.github/workflows/inspektor-gadget.yml

```
@@ -88,7 +88,9 @@ jobs:
   dependencies-checks:
     name: Dependencies checks
```

The comments make sense, thanks.
Maybe we could additionally rename the `name` and even the target to `pr-dependencies-check`
```suggestion
    name: PR dependencies checks
```